templates/block/block--system-branding-block.html.twig

```diff
@@ -0,0 +1,35 @@
+{% extends "block.html.twig" %}
```

Comment:
I can't tell what the difference between this and the stanford_basic parent template is. re: https://github.com/SU-SWS/stanford_basic/blob/8.x-1.x/templates/block/block--system-branding-block.html.twig

Does this need to be included?